gnucash/gnome/gnc-budget-view.c

```diff
@@ -1547,6 +1547,12 @@ The function will step through to only display the columns that are set
 void
 gnc_budget_view_refresh (GncBudgetView *budget_view)
 {
+	// Column identifiers
+	const gint GNC_BUDGET_VIEW_CODE_COL = 1;
+	const gint GNC_BUDGET_VIEW_DESC_COL = 2;
+	const gint GNC_BUDGET_VIEW_START_PERIODS_COL = 3;
```

Comment:
Make this an enum. There's no need to use all caps (which should be reserved for preprocessor symbols) nor to elaborate  the names with `gnc_budget_view`  in function scope.

Reply:
Done